### ISSUE TITLE
We rerun the code under the default settings and did not get the reported results

### ISSUE THREAD

**Issue by kawehburg** [bckim92/sequential-knowledge-transformer#3]
we rerun your released code, do not change at all. However, our results in 5 epoch are  as follows：
seen
{
	'rouge1': 0.16799528342797215, 
	'rouge2': 0.04995847436439046, 
	'rougeL': 0.15273983242977188, 
	'perplexity': 70.22687, 
	'total_loss': 6.5890737, 
	'accuracy': 0.25414078674948243, 
	'knowledge_loss': 2.0941563, 
	'kl_loss': 0.24318585
}
unseen
{
	'rouge1': 0.13736082976882263, 
	'rouge2': 0.02870834177474048, 
	'rougeL': 0.12473196231650303, 
	'perplexity': 110.05762, 
	'total_loss': 7.856748, 
	'accuracy': 0.12289648138704742, 
	'knowledge_loss': 2.9482818, 
	'kl_loss': 0.20746219
}
We continue to train the model to 15 epochs, and the results still has a big gap with the report and the pretrained model. We repeated the experiment many times and got similar results. This problem also exists on the holle dataset.
The best result we got on the holle dataset using released code：
{
	'rouge1': 0.2709818525254534, 
	'rouge2': 0.19968256093325662, 
	'rougeL': 0.2615125019538694, 
	'perplexity': 58.779068, 
	'total_loss': 6.9681244, 
	'accuracy': 0.24439461883408073, 
	'knowledge_loss': 2.6161168, 
	'kl_loss': 0.27822185
}
Is the pretrained model trained under the default settings?

**Comment by bckim92** [comment]
Hi, thank you for your interest in our work. We will check it as soon as possible and let you know soon.

**Comment by bckim92** [comment]
Hi, we rerun our released code and can easily get comparable results to those on our paper. As you can see in the below TensorBoard screenshot, we can get the best result at step 55.8K (epoch 6).

Since we did not make any modification at rerun, one possible reason for not getting the comparable results would be the number of GPUs, which can make a difference between your rerun and our rerun. (Real) batch size is determined by the `batchsize` and `gpus` hyperparameters. For example, if you run your code with `batchsize=1` and `gpus=0,1`, the actual batch size will be 2. If you run with `batchsize=1`and `gpus=0`, the actual batch size will be 1.

So, can you recheck your `gpus` hyperparameter? If that's not the reason, please let us know. Thank you!
![image](https://user-images.githubusercontent.com/6949363/82988918-d9bce400-a034-11ea-9064-d4066f6f93e0.png)
